feature-student/wakeup-song/src/main/java/com/b1nd/dodam/wakeupsong/WakeupSongScreen.kt

```diff
@@ -0,0 +1,369 @@
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import coil.compose.AsyncImage
+import com.b1nd.dodam.dds.component.DodamDialog
+import com.b1nd.dodam.dds.component.DodamSmallTopAppBar
+import com.b1nd.dodam.dds.component.DodamToast
+import com.b1nd.dodam.dds.component.button.DodamCTAButton
+import com.b1nd.dodam.dds.component.button.DodamLargeFilledButton
+import com.b1nd.dodam.dds.foundation.DodamIcons
+import com.b1nd.dodam.dds.style.BodyLarge
+import com.b1nd.dodam.dds.style.BodyMedium
+import com.b1nd.dodam.dds.style.LabelLarge
+import com.b1nd.dodam.dds.style.TitleLarge
+import com.b1nd.dodam.dds.style.TitleMedium
+import com.b1nd.dodam.ui.util.NoInteractionSource
+import com.b1nd.dodam.wakeupsong.model.WakeupSong
+import com.b1nd.dodam.wakeupsong.viewmodel.Event
+import com.b1nd.dodam.wakeupsong.viewmodel.WakeupSongViewModel
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalFoundationApi::class)
+@Composable
+fun WakeupSongScreen(onClickAddWakeupSong: () -> Unit, popBackStack: () -> Unit, viewModel: WakeupSongViewModel = hiltViewModel()) {
+    val uiState by viewModel.uiState.collectAsState()
+    var selectedTabIndex by remember { mutableIntStateOf(0) }
+    val snackbarHostState = remember { SnackbarHostState() }
+
+    LaunchedEffect(key1 = viewModel.event) {
+        viewModel.event.collect { event ->
+            when (event) {
+                is Event.DeleteWakeupSong -> {
+                    snackbarHostState.showSnackbar("기상송을 삭제했어요.")
+                    viewModel.getMyWakeupSong()
+                }
+            }
+        }
+    }
+
+    Scaffold(
+        modifier = Modifier
+            .fillMaxSize(),
+        topBar = {
+            DodamSmallTopAppBar(
+                title = {
+                    BodyLarge(text = "기상송")
+                },
+                onNavigationIconClick = popBackStack,
+                colors = TopAppBarDefaults.topAppBarColors(
+                    containerColor = MaterialTheme.colorScheme.background,
+                ),
+            )
+        },
+        snackbarHost = {
+            SnackbarHost(hostState = snackbarHostState) {
+                Column {
+                    DodamToast(text = it.visuals.message, trailingIcon = {
+                        Icon(imageVector = DodamIcons.Checkmark, contentDescription = "check")
```

Comment:
Please change this line to 
```kotlin
CheckmarkCircleFilledIcon(
    modifier = Modifier
            .size(20.dp)
            .drawBehind {
                drawRoundRect(
                    color = DodamColor.White,
                    topLeft = Offset(12f, 12f),
                    size = Size(30f, 30f)
                )
         }
)
```
because it's not like design